### PR TITLE
v1 support date in "REST API" articles

### PR DESCRIPTION
**Pull request by bhuvanapriyap**
Rendered article sections for review:
* **Using the REST API for updating Microsoft Edge Add-ons** > **Versions of the Update REST API**
   * https://review.learn.microsoft.com/microsoft-edge/extensions-chromium/publish/api/using-addons-api?branch=pr-en-us-3307#versions-of-the-update-rest-api
   * [Before/Live](https://learn.microsoft.com/microsoft-edge/extensions-chromium/publish/api/using-addons-api#versions-of-the-update-rest-api)
   * Updated the end date for support.
* **REST API Reference for updating Microsoft Edge Add-ons** > **Versions of the Update REST API**
   * https://review.learn.microsoft.com/microsoft-edge/extensions-chromium/publish/api/addons-api-reference?branch=pr-en-us-3307#versions-of-the-update-rest-api
   * [Before/Live](https://review.learn.microsoft.com/microsoft-edge/extensions-chromium/publish/api/addons-api-reference#versions-of-the-update-rest-api)
   * Updated the end date for support.
* **Roadmap for Microsoft Edge Add-ons** > **Deprecation of the Update REST API v1**
   * https://review.learn.microsoft.com/microsoft-edge/extensions-chromium/whats-new/roadmap?branch=pr-en-us-3307#deprecation-of-the-update-rest-api-v1
   * [Before/Live](https://learn.microsoft.com/microsoft-edge/extensions-chromium/whats-new/roadmap#deprecation-of-the-update-rest-api-v1)
   * Added h2 section.
* **Released features for Microsoft Edge Add-ons** > **Publish updates of extensions using the Microsoft Edge Add-ons API**
   * https://review.learn.microsoft.com/microsoft-edge/extensions-chromium/whats-new/released-features?branch=pr-en-us-3307#publish-updates-of-extensions-using-the-microsoft-edge-add-ons-api
   * [Before/Live](https://learn.microsoft.com/microsoft-edge/extensions-chromium/whats-new/released-features#publish-updates-of-extensions-using-the-microsoft-edge-add-ons-api)
   * Linked to Roadmap section.

AB#55065287

### CODE REVIEW

File: microsoft-edge/extensions-chromium/publish/api/using-addons-api.md [hunk 1]
```diff
@@ -30,7 +30,7 @@ To submit suggestions and feedback, enter an [Issue about the Add-ons API](https
 <!-- ------------------------------ -->
 #### Versions of the Update REST API
 
-As of September 6, 2024, both v1.1 and v1 of this Update REST API are supported.  Later, v1 will no longer be supported.  The date for ending v1 support is not yet finalized.
+As of September 6, 2024, both v1.1 and v1 of this Update REST API are supported.  Later, v1 will no longer be supported.  The date for ending v1 support is December 31, 2024.
```
Comment (mikehoffms) [inline microsoft-edge/extensions-chromium/publish/api/using-addons-api.md:33]:
```suggestion
As of September 6, 2024, both v1.1 and v1 of this Update REST API are supported.  Support for v1 will end on December 31, 2024.
```

File: microsoft-edge/extensions-chromium/publish/api/addons-api-reference.md [hunk 2]
```diff
@@ -18,7 +18,7 @@ For an overview, see [Using the REST API for updating Microsoft Edge Add-ons](./
 <!-- ------------------------------ -->
 #### Versions of the Update REST API
 
-As of September 6, 2024, both v1.1 and v1 of this Update REST API are supported.  Later, v1 will no longer be supported.  The date for ending v1 support is not yet finalized.
```
Comment (sandeepchads) [inline microsoft-edge/extensions-chromium/publish/api/addons-api-reference.md:21]: We should add -

November 2024 Update - Support for v1 will end on December 31st 2024.

Comment (mikehoffms) [reply microsoft-edge/extensions-chromium/publish/api/addons-api-reference.md:21]: in roadmap.md added-

## Deprecation of the Update REST API v1

As of September 6, 2024, both v1.1 and v1 of the Update REST API are supported.  Support for v1 will end on December 31, 2024.

See [Using the REST API for updating Microsoft Edge Add-ons](../publish/api/using-addons-api.md).

**Comment by bhuvanapriyap** [comment]
Approved - LGTM - PM

**Pull request merged**